src/components/ContextBar.js

```diff
@@ -105,16 +102,6 @@ export const ContextBar = ({
     }),
     div({ style: Style.elements.contextBarContainer }, [
       div({ style: contextBarStyles.contextBarContainer }, [
-        h(WorkspaceMenuTrigger, { canShare, isOwner, setCloningWorkspace, setSharingWorkspace, setDeletingWorkspace }, [
```

Comment:
`setCloningWorkspace`, `setSharingWorkspace`, `setDeletingWorkspace` can now be removed [above](https://github.com/DataBiosphere/terra-ui/pull/2896/files#diff-bc90e9ee24882aca9cebd73ed2d0b3e0a3acff78151dfa6893964b33f3688dd6R38).

Reply:
Resolved